core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/impl/RawInfraBuilder.kt

```diff
@@ -252,12 +252,6 @@ class RouteDescriptorImpl(
 ) : RouteDescriptor
 
 interface RestrictedRawInfraBuilder {
-    fun movableElement(
-        name: String,
-        delay: Duration,
-        init: MovableElementDescriptorBuilder.() -> Unit
-    ): TrackNodeId
```

Comment:
I'm unsure why this method is gone from the interface. Isn't there a node method now?